eureka/S3_data_reduction/plots_s3.py

```diff
@@ -206,6 +206,7 @@ def profile(eventdir, profile, submask, n, hide_plots=False):
     -------
     None
     '''
+    profile[np.where(np.isnan(profile))] = 0
```

Comment:
Zeros won't have the same impact on later steps though since things like averages will actually use those points rather than ignoring them. Wouldn't a better option would be numpy masked arrays?

Reply:
Fixed as above.